### ISSUE TITLE
I can't download the pack.

### ISSUE THREAD

**Issue by DiaDragon0818** [adam9899/Issues#427]
I just get a message saying "This modpack requires a dependency that is missing or no longer available. Missing Addon ID: 309911". I don't know what to do.

**Comment by axelFrau** [comment]
Yeah same message but not same ID "This modpack requires a dependency that is missing or no longer available. Missing Addon ID: 242064"

**Comment by ByteTemplar** [comment]
MultiMC downloaded the pack without any problems...

**Comment by ThePaleRiderz** [comment]
That is an issue with twitch

**Comment by Charismara** [comment]
Take a look on #438 

@adam9899 Please close this issue.
